src/shortcuts/deleteEmptyThoughtOrOutdent.js

```diff
@@ -21,7 +24,7 @@ import {
 } from '../selectors'
 
 // eslint-disable-next-line jsdoc/require-jsdoc
-const canExecute = getState => {
+const canExecuteDeleteEmptyThought = getState => {
```

Comment:
Subtle point, but I would recommend passing `state` directly here, as then the function better conforms to a selector signature.

Reply:
Now that I think about it, I think `shortcuts` should pass `state` directly so that `canExecute` also matches the selector signature. That doesn't have to be done in this PR.